src/scss/components/_resources.scss

```diff
@@ -83,8 +83,10 @@
 			margin-top: rem(24);
 
 			button {
+				align-items: center;

```

Comment:
It might be better to create a generic `button` style in `_base.scss` with the adjustment migrated from sanitize so it applies to all buttons across the site. There are more buttons on "Initiatives" page and "Events" page.

Reply:
I wasn't able to find the reason in detail why the children of the filter buttons ended up centre aligned under sanitize and so I thought it would be best to be conservative with this change and only apply it here. The buttons on the initiatives and events page aren't affected by this layout rule since they are simple buttons without child elements.

Reply:
Sorry, I see your point now - the other buttons have children which are the expand and contract arrows which are also slightly misaligned because of this change in layout rule. I've applied this suggestion, thanks.